product_docs/docs/migration_portal/4.0/04_mp_migrating_database/01_mp_schema_extraction/index.mdx

```diff
@@ -0,0 +1,192 @@
+---
+title: "Performing a schema extraction"
+
+legacyRedirectsGenerated:
+  # This list is generated by a script. If you need add entries, use the `legacyRedirects` key.
+  - "/edb-docs/d/edb-postgres-migration-portal/user-guides/user-guide/3.0.1/mp_schema_extraction.html"
+
+redirects:
+  - ../01_whats_new/
+
+---
+
+<div id="mp_schema_extraction" class="registered_link"></div>
+
+
+You can perform a schema extraction using either of the following methods. EDB recommends using the EDB DDL extractor to extract your schemas.
+ -  [EDB DDL extractor](#extracting-schemas-using-the-edb-ddl-extractor) (recommended method)
+ -  [Oracle’s Data Pump utility](..01_data_pump_utility/) 
```

Comment:
This link appears to be broken in the deployment version.

Reply:
@drothery-edb or @moiznalwalla Can one of you look into this and fix?

Reply:
Fixed the link.